enterprise/frontend/src/metabase-enterprise/sandboxes/components/LoginAttributesWidget.tsx

```diff
@@ -17,13 +17,20 @@ export const LoginAttributesWidget = ({
   className,
   style,
 }: Props) => {
-  const [{ value }, , { setValue }] = useField(name);
+  const [{ value }, , { setValue, setError }] = useField(name);
+
+  const handleError = (error: boolean) => {
```

Comment:
I don't love how this is done, but I wasn't sure what else to do. Errors are cleared after setValue is called, so inside <MappingEditor /> I need to ensure that `onChange` isn't actually called if an error is detected